docs/integration/index.md

```diff
@@ -473,6 +473,108 @@ target="_blank" rel="noopener">another AWS Lambda function</a>. If
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/react.png" alt="React" title="React" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>React</h5>Full support for single-page applications built with the React library.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+	<div class="mdl-cell mdl-cell--4-col">
+		<a href="../experience/integrations#vuejs">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/vuejs.png" alt="Vue.js" title="Vue.js" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Vue.js</h5>Full support for single-page applications built with Vue.js.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+	<div class="mdl-cell mdl-cell--4-col">
+		<a href="../experience/integrations#ember">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/ember.png" alt="Ember" title="Ember" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Ember</h5>Full support for single-page applications built with the Ember framework.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+  <div class="mdl-cell mdl-cell--4-col">
+		<a href="../experience/integrations#static-websites">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/static.png" alt="Static Websites" title="Static Websites" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Static Websites</h5>Full support for all types of static websites.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+  <div class="mdl-cell mdl-cell--4-col">
+		<a href="../experience/integrations#server-side-rendered-websites">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/server.png" alt="Server-side rendered websites" title="Server-side rendered websites" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Server-side rendered websites</h5>Full support for all types of SSR websites and applications.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+</div>
+
+All of the integrations require adding and [configuring the Experience script](../experience/getting-started). However, single-page applications require one more configuration step to register route changes.
```

Comment:
Should we add an integration for "static websites"?

Reply:
I'd keep it here as part of the Experience integrations. What's your take on it? We can remove the SSR and Static Websites if you think they're not relevant.

Reply:
I'm not sure what you are commenting on here.  I was asking if you think we should mention that this integrates with static websites, too, not just React, Vue.js, and friends.

Reply:
Oh! I didn't understand what you meant then. Yes, above this paragraph I've added the cards and Static Websites is among them. Check it out. :point_down: 
![Selection_534](https://user-images.githubusercontent.com/15029531/63151750-79f7ce00-c00a-11e9-85b8-853066632220.png)